### ISSUE TITLE
Custom BG reverting back to album cover

### ISSUE THREAD

**Issue by zsaf25** [Astromations/Hazy#52]
Hi I tried applying the custom background and it works but sometimes switching songs will revert it back to the album cover. Any help? TIA 
![Screenshot 2024-01-24 135222](https://github.com/Astromations/Hazy/assets/141448669/dcaa99d1-0fae-45f9-b19c-e4373a00455e)
![Screenshot 2024-01-24 135330](https://github.com/Astromations/Hazy/assets/141448669/f4ad7be4-ecba-477d-b1dd-76c02bf0fdda)

**Comment by Astromations** [comment]
I'm not getting the issue on my end so let's run through your specs.
1. What's your OS
2. Do you have Hazy installed through marketplace or on your system
3. What other extensions/themes do you have (you can SS your "Installed" page if it'd be tedious to list them)
4. What's your Spotify and Spicetify versions | You can find that here:
![image](https://github.com/Astromations/Hazy/assets/80211195/83bae3b4-f62b-4c24-9916-82ff36ae19d6)

**Comment by zsaf25** [comment]
Hey, I believe I might have fixed it by reinstalling Spicetify, removing current theme in the config-xpui, apply the config and re-applying the theme via marketplace. 

But if you're still interested in trying to replicate the issue I ran into;

1. I'm on Windows 11 22H2
2. I installed the theme via manual installation  (downloading and placing the 3 files into the themes folder under Hazy) as installing the theme via marketplace didn't work and just applies a black theme. 
3. I only have Full App Display and Lyrics Plus installed (installed them before installing the theme)
4. 
![Screenshot 2024-01-25 125448](https://github.com/Astromations/Hazy/assets/141448669/788026c0-1a4f-4fcf-b41b-e27af2ed8ca0)
![Screenshot 2024-01-25 125622](https://github.com/Astromations/Hazy/assets/141448669/b68a949f-a248-466c-9869-f7b8b088192b)
